service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -182,6 +185,7 @@
     private final FurtherDirections orderFurtherDirections;
     private final GeneratedOrder order;
     private final Integer orderMonths;
+    private final InterimEndDate interimEndDate;
```

Comment:
Do we not need a `@Valid` annotation?

Reply:
No as I am directly passing the interimDate object instead of the whole `CaseData` object to the validator